mica/archive/aca_dark/dark_cal.py

```diff
@@ -11,10 +11,11 @@
 from astropy.io import fits, ascii
 from astropy.table import Column
 import pyyaks.context
-from Chandra.Time import DateTime
+from cxotime import CxoTime
 from chandra_aca.aca_image import ACAImage
 
-from chandra_aca.dark_model import dark_temp_scale, DARK_SCALE_4C
+from chandra_aca.dark_model import dark_temp_scale
+from chandra_aca.dark_model import DARK_SCALE_4C  # noqa
```

Comment:
Is this a new convention or?

Reply:
I added the `# noqa` statement so flake doesn't complain about `DARK_SCALE_4C` being defined and not used, because I figured we still want to keep it so client code can use it as `dark_cal.DARK_SCALE_4C`.

I moved it on its own line so it is obvious what is it that flake is supposed to ignore. In other words, the import of dark_temp_scale should not be ignored by the linter, and if we ever remove the code that uses it flake should complain about the unused symbol.

Reply:
Yet it was not obvious to me, as I wasn't paying attention to the original issue.... Not sure if we should leave it here for client code or not at this point then.  It is a good question.